langchain_openai_api_bridge/fastapi/chat_completion_router.py

```diff
@@ -16,10 +16,11 @@
 
 def create_chat_completion_router(
     tiny_di_container: TinyDIContainer,
+    custom_event_handler: callable = lambda event: None,
 ):
-    chat_completion_router = APIRouter(prefix="/chat/completions")
+    chat_completion_router = APIRouter(prefix="/chat")
 
-    @chat_completion_router.post("/")
+    @chat_completion_router.post("/completions")
```

Comment:
I meant to mention I included this unrelated change. In my case I had to include the following for the Vercel SDK to successfully hit my custom endpoint:
```
# Remove trailing slashes from routes, because LangchainOpenaiApiBridgeFastAPI adds them
for route in app.routes:
    if route.path.endswith("/"):
        app.routes.remove(route)
        app.add_api_route(route.path[:-1], route.endpoint, methods=route.methods)
```
This seems to resolve that without causing side effects.

Reply:
Make sure Integration Tests are all passing. If this is the case, everything should be fine.